app/src/main/res/layout/topic_train_skill_view.xml

```diff
@@ -15,18 +15,28 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="wrap_content"
+    app:barrierDirection="right"
+    app:constraint_referenced_ids="button_example,text_view_status">
+
+    <androidx.constraintlayout.widget.Barrier
+      android:id="@+id/barrier"
```

Comment:
Let's use an identifier that's more specific, like ``topic_train_skill_barrier`` or similar.

Reply:
Done.